schema/spacewalk/common/tables/suseKiwiProfile.sql

```diff
@@ -19,7 +19,8 @@ CREATE TABLE suseKiwiProfile
                   CONSTRAINT suse_kiwi_prid_fk
                      REFERENCES suseImageProfile (profile_id)
                      ON DELETE CASCADE,
-    path        VARCHAR(1024) NOT NULL
+    path        VARCHAR(1024) NOT NULL,
+    kiwi_options VARCHAR(256)
```

Comment:
Just wondering, Is 256 enough for potentially lengthy options? Considering this is not only for profiles, but generic options.